README.md

```diff
@@ -1,3 +1,6 @@
 ### DSA- Java
+
+[![Join the chat at https://gitter.im/DSA-Java/community](https://badges.gitter.im/DSA-Java/community.svg)](https://gitter.im/DSA-Java/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
I will be updating README now. Just add this button on that replacing my default button